tests/test_Perceptron.py

```diff
@@ -30,22 +30,20 @@ def perceptron():
 
 
 def test_calc_activity(inputs, perceptron):
-    perceptron.calc_activity(inputs)
+    perceptron.calc_activity(inputs, bias=perceptron.bias)
```

Comment:
I removed the bias bool calc_activity() and set it to determine if it should update the bias based on how the Perceptron object is instantiated